### PR TITLE
Explore Africa project Procedure

### PR DESCRIPTION
**Pull request by Josphat205**
## The procedure of project development

- Created react app

- Setup redux in the project

- Added all necessary components

- Fetched data from an API using createAsyncThunk

- Displayed data in the UI by dispatching

- Implemented search technique

[Live Presentation](https://www.loom.com/share/6f39e75b7f074c4d8390fe79d6488700)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ Explore-african-countries-project is a web application that fetches data for all
...
-[Explore-African-Countries](https://explore-africa-loman.netlify.app/)
-
 ## Live Presentation
 
 [Explore-African-Countries](https://www.loom.com/share/6f39e75b7f074c4d8390fe79d6488700)
 
+## Demo
+
+[Explore-African-Countries](https://explore-africa-loman.netlify.app/)
+
```
Comment (codepantha) [inline README.md:15]: - [x] It is essential that your README contains an "Acknowledgements" section that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the author of the [original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 

## Hi @Josphat205 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Highlights 🎯 
✔️ Descriptive PR
✔️ Well-explained demo video
✔️ Your app works well
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] You have written a few tests 👏 but you need to write more unit tests for your app's components such as `CountryComponent`, `CountryLists` etc in order to have more coverage 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -59,6 +59,9 @@ Contributions, issues, and feature requests are welcome!
 
 Give a ⭐ if you like this project!
 
+## Acknowledgement
+Acknowledgments Great appreciation to Microverse community. Much appreciate to Nelson Sakwa Createor of design 📝 License This project is MIT licensed.
+
```
Comment (codepantha) [inline README.md]: - [ ]  Well done for acknowledging Nelson Sakwa. I recommend you link back to the [original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) so anyone can easily see where you got your design inspiration thereby giving the author appropriate credit. This is the essence of the acknowledgement 😎

**Review by codepantha — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️ 

### Hi @Josphat205 👋, 

Great work on making the changes requested previously 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Josphat205 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the requested changes from the previous reviewers

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**